js/script.js

```diff
@@ -13,20 +13,39 @@
  **/
 
 function Generator() {
-
     /* Name Arrays: Customize names to change possible output */
+  this.initialize = function () {
+    document.getElementById("enter").addEventListener("click", this.createRapName);
+  }
+
+  this.createRapName = function () {
```

Comment:
It's much better practice when you initialize functions for objects, to declare them outside of the object. IE. 

```Generator.prototype.createRapName = function() {...```

This prevents the code to have to be reinitialized everytime you want to call it. A small memory loss here, but it can stack up in bigger files.